media-proxy/src/rdma_session.c

```diff
@@ -314,7 +314,7 @@ static void rx_rdma_consume_frame(rx_rdma_session_context_t *s, char *frame)
     uint32_t buf_size = s->transfer_size;
     uint16_t rx_buf_num = 0, rx = 0;
 
-    if (!s->shm_ready) {
+    if (!atomic_load(&s->shm_ready)) {
         INFO("%s memif not ready\n", __func__);
         return;
     }
```

Comment:
Move this check to the calling function.
```suggestion
```

Reply:
done